StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/Vertical Saved Payment Method Screen/VerticalSavedPaymentMethodsViewController.swift

```diff
@@ -131,7 +131,7 @@ class VerticalSavedPaymentMethodsViewController: UIViewController {
     }()
 
     private lazy var stackView: UIStackView = {
-        let stackView = UIStackView(arrangedSubviews: [headerLabel] + paymentMethodRows)
+        let stackView = UIStackView(arrangedSubviews: [headerLabel] + paymentMethodRows + [UIView.makeSpacerView()])
```

Comment:
I'm worried the spacerview will have ambiguous height. I think what you really want is a view that tries to be zero height but at low priority.